girder/utility/s3_assetstore_adapter.py

```diff
@@ -473,7 +473,7 @@ def fileUpdated(self, file):
         disp = 'attachment; filename="%s"' % file['name']
         mime = file.get('mimeType') or ''
 
-        if key['ContentType'] != mime or key['ContentDisposition'] != disp:
+        if key.get('ContentType') != mime or key.get('ContentDisposition') != disp:
```

Comment:
I'm a little unsure about this change.  These keys are now missing in the `server_file` test, but I'm not certain why.